src/Monolog/Formatter/LogzIoFormatter.php

```diff
@@ -0,0 +1,50 @@
+<?php declare(strict_types=1);
+
+/*
+ * This file is part of the Monolog package.
+ *
+ * (c) Jordi Boggiano <j.boggiano@seld.be>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Monolog\Formatter;
+
+/**
+ * Encodes message information into JSON in a format compatible with Logz.io.
+ *
+ * @author Christian Brückner <chris@chrico.info>
+ */
+class LogzIoFormatter extends JsonFormatter
+{
+
+    /**
+     * yyyy-MM-dd'T'HH:mm:ss.SSSZ
+     */
+    const DATETIME_FORMAT = "c";
```

Comment:
Howdy!

As far as i know, the current used time library for date parsing in Logstash and ElasticSearch are only supporting millisecond precision (http://www.joda.org/joda-time/). So there is no support for microseconds.
But as they mentioned in the article above, it's possible to add other date formats as well. Since this is a customization, it shouldn't be handled by the default Handler/Formatter in Monolog.

The [integration in Python](https://github.com/logzio/logzio-python-handler/blob/master/logzio/handler.py#L34) also uses `now.microsecond / 1000` (= milliseconds). Just the NodeJS-integration above has a additional field `@timestamp_nano` which is not offical supported by default.